hack/arktos-up-scale-out-poc.sh

```diff
@@ -681,6 +683,49 @@ if [ "${IS_RESOURCE_PARTITION}" == "true" ]; then
 fi
 
 if [ "${IS_RESOURCE_PARTITION}" != "true" ]; then
+  # Applying mizar cni
+  if [[ "${CNIPLUGIN}" == "mizar" ]]; then
+    ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create configmap system-source --namespace=kube-system --from-literal=name=arktos --from-literal=company=futurewei
+
+    # Creating mizar crds
+    echo "Creating mizar crds ......."
+    cp "${KUBE_ROOT}/third_party/mizar/mizar-crds.yaml" mizar-crds.yaml
+    ${KUBECTL} apply -f mizar-crds.yaml
+    rm mizar-crds.yaml
+
+    if [[ "${IS_SECONDARY_TP}" == "false" ]]; then
```

Comment:
when IS_SECONDARY_TP == true? what is the purpose for this condition?
How about 3rd, 4th... TP?

Reply:
> when IS_SECONDARY_TP == true? what is the purpose for this condition? How about 3rd, 4th... TP?

Aside from primary TP, which will have mizar daemonset deployed, all other TPs are secondardy TP.